### ISSUE TITLE
After install this is notice messages

### ISSUE THREAD

**Issue by dannythebestguy** [soflyy/oxygen-bugs-and-features#1353]
Notice: Trying to get property 'term_id' of non-object in /home/dannythebestguy/sites/homepageideas/current/public/wp-content/plugins/oxygen/component-framework/components/classes/menu-pro.class.php on line 167

Notice: Trying to get property 'name' of non-object in /home/dannythebestguy/sites/homepageideas/current/public/wp-content/plugins/oxygen/component-framework/components/classes/menu-pro.class.php on line 167

Notice: Trying to get property 'term_id' of non-object in /home/dannythebestguy/sites/homepageideas/current/public/wp-content/plugins/oxygen/component-framework/components/classes/menu-pro.class.php on line 167

Notice: Trying to get property 'name' of non-object in /home/dannythebestguy/sites/homepageideas/current/public/wp-content/plugins/oxygen/component-framework/components/classes/menu-pro.class.php on line 167

Notice: Undefined index: button_priority in /home/dannythebestguy/sites/homepageideas/current/public/wp-content/plugins/oxygen/component-framework/api/oxygen.element.class.php on line 38

**Comment by KittenCodes** [comment]
@dannythebestguy 

> Notice: Trying to get property 'term_id' of non-object in /home/dannythebestguy/sites/homepageideas/current/public/wp-content/plugins/oxygen/component-framework/components/classes/menu-pro.class.php on line 167

> Notice: Trying to get property 'name' of non-object in /home/dannythebestguy/sites/homepageideas/current/public/wp-content/plugins/oxygen/component-framework/components/classes/menu-pro.class.php on line 167

Do you know the specific steps needed to cause these two errors?

**Comment by wpsumo** [comment]
@KittenCodes Once I copied shortcodes from one site to another it caused following:

`Notice | Trying to get property 'ID' of non-object | 1 | +wp-content/plugins/oxygen/component-framework/components/classes/reusable.class.php:55 | Plugin: oxygen`

1. Copy shortcode from one template on one site.
2. Paste shortcode from step 1 into a template in the new site and save.
3. Reassign shortcodes
4. Regenerate css
5. Open the template on the new site and save the tample.
6. Issue and php notice still there.

**Comment by KittenCodes** [comment]
@wpsumo Are you able to send me the shortcodes you're copying please?

**Comment by wpsumo** [comment]
I shared it over support.

**Comment by KittenCodes** [comment]
> I can share over support ticket. But you refer to the shortcode template?

Yes please - if you can paste the shortcodes here that are causing this error, that would be great.

**Comment by wpsumo** [comment]
Due to privacy and code sharing publicly I will not. I shared in support ticket.

**Comment by mindfullsilence** [comment]
Happening to me as well. Same error when creating a tamplate that uses the menu pro component.

**Comment by KittenCodes** [comment]
Thank you @mindfullsilence. I've updated our internal bug report.
